sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Config/ServiceBusHostBuilderExtensionsTests.cs

```diff
@@ -197,12 +182,12 @@ public void ReadDefaultConnectionString(string defaultConnectionString, string s
                 var envPrpvider = (test.ImplementationInstance as ConfigurationRoot).Providers
                     .Single(x => x.GetType() == typeof(EnvironmentVariablesConfigurationProvider));
                 envPrpvider.Set("ConnectionStrings:" + Constants.DefaultConnectionStringName, defaultConnectionString);
-                envPrpvider.Set(Constants.DefaultConnectionSettingStringName, sefaultConectionSettingString);
+                envPrpvider.Set(Constants.DefaultConnectionSettingStringName, defaultConectionSettingString);
 
                 b.AddServiceBus();
             }, new Dictionary<string, string>());
 
-            Assert.AreEqual(options.ConnectionString, expectedValue);
+            //Assert.AreEqual(options.ConnectionString, expectedValue);
```

Comment:
What's the point of this test?

Reply:
the connection string is no longer included in the options, but we can at least assert that the options instance is not null.

Reply:
I added an equivalent test.